coral/coral/templates/views/components/widgets/cover-letter-widget.htm

```diff
@@ -11,11 +11,12 @@
     <h5 class="summary-value" data-bind="text: label"></h5>
     <!-- /ko -->
     <div class="col-xs-12">
+      
       <div
         style="
           border: 5px solid #eee;
           padding: 10px;
-          max-width: 500px;
+          max-width: 800px;
```

Comment:
Cover letter changes

Reply:
Sorry, hidden in with the version up commits, must of had unpushed commits when i  returned to version up